soc/arm/gigadevice/gd32f4xx/Kconfig.defconfig.gd32f407

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@gmail.com>
```

Comment:
Please consider to split this commit to `dts:` and `soc:`.

Reply:
Thank you for reviewing.
Fixed it.